curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e230500602983e01fff6e.md

```diff
@@ -21,53 +21,93 @@ Your `checkForStraights` variable should be a function.
 assert.isFunction(checkForStraights);
 ```
 
-If a small straight is rolled, your `checkForStraights` function should enable the fourth radio button, set the value to `30`, and update the displayed text to `, score = 30`.
+If a small straight is rolled, your `checkForStraights` function should enable the fourth radio button, set the value to `30`, update the displayed text to `, score = 30` and leave the fifth radio button disabled.
```

Comment:
do not change how the step deals with the 5th radio button, that is dealth with in the new step 15
this function should update the 5th radio button